docs/modules/ROOT/pages/index.adoc

```diff
@@ -242,14 +243,31 @@ The content of `signature` depends on the type of credential that is used. Valid
 
 The *<<_identity_assertion,identity assertion>>* shall have a label of `cawg.identity`. More than one *<<_identity_assertion,identity assertion>>* may appear in a _<<C2PA claim>>;_ if this occurs, these assertions shall be given unique labels as described by link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_multiple_instances++[Section 6.4, “Multiple instances,”] of the C2PA technical specification.
 
-[#describe-role-in-content]
-NOTE: TO DO (link:https://github.com/creator-assertions/identity-assertion/issues/14[issue #14]): Describe how to allow actors to describe their role in the content (previously described via CreativeWork assertion) and potentially in specific steps referenced in C2PA actions assertion.
-
 [#describe-social-media]
 NOTE: TO DO (link:https://github.com/creator-assertions/identity-assertion/issues/15[issue #15]): Describe where would things like social media accounts live.
 
-[#describe-actor-role]
-NOTE: TO DO (link:https://github.com/creator-assertions/identity-assertion/issues/26[issue #26]): Describe credential holder’s role in relation to the asset.
+==== Credential subject roles
+
+The _<<_credential_holder,credential holder>>_ MAY choose to describe the relationship of the _<<_credential_subject,credential subject>>_ to the _<<C2PA asset>>_ by adding a `role` field to `signer_payload`.
+
+The following values for `role` are defined. It is recommended that the value of `role` be one of the following values and that any user experience that presents the content of an *<<_identity_assertion,identity assertion>>* be prepared to describe these roles in the same human language as the surrounding interface:
```

Comment:
Consider moving this into its own standard?

Reply:
Per Nigel comment, do we need to allow multiple roles per named actor?

Reply:
Above feedback was from last Monday (29 April)'s meeting. I'm planning to push back on that in the interest of keeping this specification simpler. I think we _could_ add a JSON-LD vocabulary here in a subsequent release if there's significant pushback.